pkg/aws/sts/metrics.go

```diff
@@ -0,0 +1,61 @@
+package sts
+
+import "github.com/prometheus/client_golang/prometheus"
+
+var (
+	cacheHit = prometheus.NewCounter(
+		prometheus.CounterOpts{
+			Namespace: "kiam",
+			Subsystem: "sts",
+			Name:      "cache_hit_total",
+			Help:      "Number of cache hits to the metadata cache",
+		},
+	)
+
+	cacheMiss = prometheus.NewCounter(
+		prometheus.CounterOpts{
+			Namespace: "kiam",
+			Subsystem: "sts",
+			Name:      "cache_miss_total",
+			Help:      "Number of cache misses to the metadata cache",
+		},
+	)
+
+	errorIssuing = prometheus.NewCounter(
+		prometheus.CounterOpts{
+			Namespace: "kiam",
+			Subsystem: "sts",
+			Name:      "error_issuing_count",
```

Comment:
To match the other metrics could we rename to `issuing_errors` or `issuing_errors_total` please?